subsys/bluetooth/controller/hci_internal.c

```diff
@@ -1055,14 +1058,14 @@ static uint8_t le_controller_cmd_put(uint8_t const * const cmd,
 			sizeof(sdc_hci_cmd_le_set_transmit_power_reporting_enable_return_t);
 		return sdc_hci_cmd_le_set_transmit_power_reporting_enable((void *)cmd_params,
 									  (void *)event_out_params);
+#endif
 
 	case SDC_HCI_OPCODE_CMD_LE_READ_RF_PATH_COMPENSATION:
```

Comment:
What ifdefs should be put around the RF path compensation commands?

Reply:
It's unconditional.  It's not LL feature and not SDC feature.

Reply:
The spec lists these commands with 
"C.22: Mandatory if the LE Controller supports sending Transmit Power in advertisements
or if LE Feature (LE Power Control Request) is supported, otherwise optional."
See Core_v5.4, Vol 4, Part E, Section 3.

I would therefore expect there to be an ifdef for advertising extensions or LE power control.

Reply:
It does not say "otherwise excluded", right? We decided to support it unconditionally. It does not really depend on any other features, it's other way around: power control depends on it. We may decide to change it as well, and include it only if required by Power control.

Reply:
It ould be nice to document this decision in the commit message